simulator/Executive.ts

```diff
@@ -42,14 +43,19 @@ export class Executive implements IExecutive {
         this.maxTasks = options.maxTasks;
     }
 
-    private publishTaskCount(warriorId: number, taskCount: number) {
+    private publishTaskCount(warrior: IWarrior, taskCount: number) {
```

Comment:
This actually makes my life a lot easier on the frontend as I can write retrieval function based on warrior properties not just the id